geniesp/bpc_config.py

```diff
@@ -9,14 +9,16 @@
 class Brca(BpcProjectRunner):
     """NSCLC BPC sponsored project"""
 
-    # Sponsorted project name
+    # Sponsored project name
     _SPONSORED_PROJECT = "BrCa"
     # Redcap codes to cbioportal mapping synid and form key is in
     _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.7"
     # Mapping from Synapse Table to form (derived files)
     _DATA_TABLE_IDS = "syn22296821"
     # Storage of not found samples
     _SP_REDCAP_EXPORTS_SYNID = "syn21446571"
+    # MAF file version (8.1-public)
+    _SP_MAF = "syn22228700"
```

Comment:
ideally, we just have the release version here.  We can have a function that gets all the respective files and creates a mapping of the synapse ids.  This is because all the other synapse ids for all the file formats are hardcoded.

Reply:
Heres a function from the private repo: https://github.com/Sage-Bionetworks/Genie_processing/blob/3bed8e65030b9160b09d12c063108e11341b940c/utility_scripts/consortium_to_bpc.py#L11-L22

Reply:
I am a little wary of adding a dependency in this codebase to a yet another external mapping table.  It's already time consuming to debug because of tracking to all the existing dependencies.  I think it would be more transparent to leave the explicit Synapse ID in configuration file.

Reply:
The problem is we would need to update all the other synapse ids for the fusion, CNA, etc etc, since they all use the public release

Reply:
Ok, glad you pointed that out.  I didn't see the other Synapse IDs.  I'll try and implement with the folder ID